apps/api/src/app/events/usecases/send-message/send-message-email.usecase.ts

```diff
@@ -417,6 +422,17 @@ export class SendMessageEmail extends SendMessageBase {
       return;
     }
   }
+
+  public static buildFactoryIntegration(integration: IntegrationEntity, senderName?: string) {
+    return {
+      ...integration,
+      credentials: {
+        ...integration.credentials,
+        senderName: senderName && senderName.length > 0 ? senderName : integration.credentials.senderName,
```

Comment:
```suggestion
        senderName: senderName?.length > 0 ? senderName : integration.credentials.senderName,
```
Just saving characteres.

Reply:
Doing this typescript compliance about senderName possible being undefined...

Reply:
You are right. I missed that for not trying the PR in my local.